src/library/assert-slot.test.ts

```diff
@@ -38,6 +38,25 @@ class WhenNotUsedInsideATag extends LitElement {
   }
 }
 
+@customElement('glide-core-reslotted')
```

Comment:
Not sure if "reslotted" used here and throughout is descriptive enough. Open to suggestions.